build.gradle.kts

```
@@ -0,0 +1,157 @@
+@file:Suppress("SpellCheckingInspection")
+
+import org.gradle.api.JavaVersion.VERSION_11
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("jvm") version "1.6.21"
+    `java-library`
+    `maven-publish`
+    signing
+    id("org.owasp.dependencycheck") version "7.3.0"
+    id("io.github.gradle-nexus.publish-plugin") version "1.0.0"
+}
+
+group = "com.exactpro.th2"
+version = project.findProperty("release_version") ?: "1.0-SNAPSHOT"
+
+java {
+    sourceCompatibility = VERSION_11
+    targetCompatibility = VERSION_11
+
+    withJavadocJar()
+    withSourcesJar()
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+    maven {
+        name = "Sonatype_snapshots"
+        url = uri("https://s01.oss.sonatype.org/content/repositories/snapshots/")
+    }
+    maven {
+        name = "Sonatype_releases"
+        url = uri("https://s01.oss.sonatype.org/content/repositories/releases/")
+    }
+
+    configurations.all {
+        resolutionStrategy.cacheChangingModulesFor(0, "seconds")
+        resolutionStrategy.cacheDynamicVersionsFor(0, "seconds")
+    }
+}
+
+dependencies {
+    api(platform("com.exactpro.th2:bom:4.0.2"))
+
+    implementation("com.exactpro.th2:grpc-common:4.0.0-TH2-4262-reduce-load-book-and-page-3574839736-SNAPSHOT")
+    implementation("com.exactpro.th2:common:4.0.0-TH2-4262-reduce-load-book-and-page-3574992387-SNAPSHOT")
+    implementation("com.exactpro.th2:common-utils:0.0.1-book-and-page-3572891451-SNAPSHOT")
+    implementation("com.exactpro.th2:grpc-lw-data-provider:2.0.0-th2-3834-refactor-3562700951-SNAPSHOT")
+
+    implementation("com.fasterxml.jackson.core:jackson-core")
+    implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
+    implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml")
+
+    testImplementation("org.jetbrains.kotlin:kotlin-test-junit5")
+    testImplementation("org.mockito.kotlin:mockito-kotlin:4.0.0")
+
+    testImplementation("org.junit.jupiter:junit-jupiter-api:5.9.0")
+    testImplementation("org.junit.jupiter:junit-jupiter-params:5.9.0")
+    testRuntimeOnly("org.junit.jupiter:junit-jupiter:5.9.0")
```

Can be replaced with a single line
```suggestion
    testImplementation("org.junit.jupiter:junit-jupiter:5.9.0")
```